WORKSPACE

```diff
@@ -21,3 +21,9 @@ tf_workspace1()
 load("@//tensorflow:workspace0.bzl", "tf_workspace0")
 
 tf_workspace0()
+
+new_local_repository(
+    name = "webnn_native_project",
+    path = "/home/wm/work/webml/webnn/webnn-native",
```

Comment:
Hard code here. Need to find a way to reference `webnn-native`.

Reply:
It is probably OK to let user to add the local repo of webnn-native. Is there any ways to supply it via command line?

Reply:
There's an example here https://stackoverflow.com/questions/65396733/bazel-setting-a-global-varible-from-environment-variable, I will try later.